SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -524,7 +542,7 @@ void CNEMOEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contai
   for (iMarker = 0; iMarker < geometry->GetnMarker(); iMarker++) {
     if ((config->GetMarker_All_KindBC(iMarker) != INTERNAL_BOUNDARY) &&
         (config->GetMarker_All_KindBC(iMarker) != PERIODIC_BOUNDARY)) {
-
+    
       SU2_OMP_FOR_STAT(OMP_MIN_SIZE)
```

Comment:
Check if your IDE has an option to remove trailing blank spaces, there are also scripts for that purpose in the repo.

Reply:
Yep, I saw you already removed some trailing whitespaces in #1167 for the NEMO part😉

Reply:
The IDE removed them for me, when I fixed the compiler warnings.